modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/metrics/StdoutReporter.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021-2021 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.rdbloader.dsl.metrics
+
+import com.snowplowanalytics.snowplow.rdbloader.common.config.Config
+import com.snowplowanalytics.snowplow.rdbloader.dsl.Logging
+
+object StdoutReporter {
+
+  def build[F[_]](stdoutConfig: Config.Metrics.Stdout, logging: Logging[F]): Reporter[F] = 
+    new Reporter[F] {
+      def report(metrics: List[Metrics.KVMetric]): F[Unit] = {
+        val formatted = metrics.map(formatMetric(stdoutConfig.prefix))
+        val metricsStr = s"METRICS - ${formatted.mkString(" ")}"
```

Comment:
Do you think there is a need to make stdout formatting similar between Enrich and rdb loader?

Just FYI in Enrich the stdout metrics look like this

```
[ioapp-compute-1] INFO enrich.metrics - snowplow.enrich.raw = 20000
[ioapp-compute-1] INFO enrich.metrics - snowplow.enrich.good = 20000
[ioapp-compute-1] INFO enrich.metrics - snowplow.enrich.bad = 0
```

where...
- `[ioapp-compute-1] INFO enrich.metrics -` is the prefix added automatically by our slf4j configuration
- `enrich.metrics` is what we choose to call the slf4j logger
- `snowplow.enrich.raw = 20000` is what we send to the logger.
 
Note the spaces around the `=`, and one line per metric.

Reply:
Good point !

We don't use slf4jlogger in RDB loader so it won't be exactly the same but I will also use one line per metric with the ` = `.

Reply:
Actually I think that the time has come to add slf4jlogger to RDB loader